sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/implementation/WebPubSubsImpl.java

```diff
@@ -57,256 +52,205 @@
-                @BodyParam("text/plain") String message,
+                @BodyParam("text/plain") BinaryData message,
+                RequestOptions requestOptions,
                 Context context);
 
         @Head("/api/hubs/{hub}/connections/{connectionId}")
-        @ExpectedResponses({200, 404})
-        @UnexpectedResponseExceptionType(HttpResponseException.class)
         Mono<Response<Boolean>> connectionExists(
                 @HostParam("$host") String host,
                 @PathParam("hub") String hub,
                 @PathParam("connectionId") String connectionId,
-                @QueryParam("api-version") String apiVersion,
+                RequestOptions requestOptions,
                 Context context);
 
         @Delete("/api/hubs/{hub}/connections/{connectionId}")
-        @ExpectedResponses({200})
-        @UnexpectedResponseExceptionType(HttpResponseException.class)
         Mono<Response<Void>> closeClientConnection(
                 @HostParam("$host") String host,
                 @PathParam("hub") String hub,
                 @PathParam("connectionId") String connectionId,
-                @QueryParam("reason") String reason,
-                @QueryParam("api-version") String apiVersion,
+                RequestOptions requestOptions,
                 Context context);
 
         @Post("/api/hubs/{hub}/connections/{connectionId}/:send")
-        @ExpectedResponses({202})
-        @UnexpectedResponseExceptionType(HttpResponseException.class)
         Mono<Response<Void>> sendToConnection(
                 @HostParam("$host") String host,
                 @PathParam("hub") String hub,
                 @PathParam("connectionId") String connectionId,
-                @QueryParam("api-version") String apiVersion,
-                @HeaderParam("Content-Type") WebPubSubContentType contentType,
-                @BodyParam("application/octet-stream") Flux<ByteBuffer> message,
+                @HeaderParam("Content-Type") String contentType,
+                @BodyParam("application/octet-stream") BinaryData message,
                 @HeaderParam("Content-Length") long contentLength,
+                RequestOptions requestOptions,
                 Context context);
 
         @Post("/api/hubs/{hub}/connections/{connectionId}/:send")
-        @ExpectedResponses({202})
-        @UnexpectedResponseExceptionType(HttpResponseException.class)
         Mono<Response<Void>> sendToConnection(
                 @HostParam("$host") String host,
                 @PathParam("hub") String hub,
                 @PathParam("connectionId") String connectionId,
-                @QueryParam("api-version") String apiVersion,
-                @BodyParam("text/plain") String message,
+                @BodyParam("text/plain") BinaryData message,
+                RequestOptions requestOptions,
                 Context context);
 
         @Head("/api/hubs/{hub}/groups/{group}")
-        @ExpectedResponses({200, 404})
```

Comment:
Was this meant to be removed? Without this the API will throw an error when 404 is returned. Previously it would have returned without an error.

Reply:
For LLC, we don't return a strongly typed response model. It will always be the complete JSON payload with error codes.